### ISSUE TITLE
Great Hammer changes

### ISSUE THREAD

**Issue by Nopebody** [Dahoom69/Champions-Ashes-Dev#995]
**Ledo**

Rock mode FR2 is a straight downgrade in practical use compared to the generic/non-rock mode FR2. I suggest either buffing the rock mode FR2's (both 1h and 2h) tracking/active frames to be comparable, or just give it the generic GH FR2 when in rock mode. 

Also, the rock mode FR2's seem to have way less recovery than they should, maybe as a side effect of how Gabri sped them up a long time ago, which I think should be nerfed. 

Reduce the stamina cost of WA R2, it shouldn't use so much stamina for how slow and impractical it is. It'll leave you in negative stamina unless you have near 40 endurance which is absurd. 

Running R1 > R2 feint is useless because of it's unique R2. I suggest making Ledo's running r1 > r2 feint into the second R2 (both in rock mode and standard mode). 

In summary: 

- [x] Buff or replace rock mode Fr2 
- [x] Nerf rock mode Fr2 recoveries (if you decide to keep it) 
- [x] Reduce WA R2 stamina cost 
- [x] Add feint limit to WA R2 (bug, can still do WA R2 > R2 > R1/R2) 
- [x] Make Running R1 > R2 feint to Second R2 (only for ledo, not the other great hammers) 

**Quakestone**

- [x] Give WA > R2 feint back (not extension skip, the actual R2 feint).

I know this got removed a long time ago, but that was before we had the feint limit so it was removed because of WA > R2 > R1 stuff, which isn't a thing anymore. It'll really help the move out, because all it can do right now is feint to R1 which kind of sucks because of the limited range of great hammer R1s. 

Might update this as I notice or think of stuff but that's all for now.
